.github/workflows/docker_build_tpls.yml

```diff
@@ -25,7 +25,9 @@ jobs:
       id: dockerfile_name
     - name: Get dockerhub repo name
       shell: bash
-      run: echo "##[set-output name=repo_plus_tag;]$(echo ${REPO}:${TAG})"
+      run: |
+        echo ${REPO}:${TAG}-`date "+%m-%d-%y-%Hh-%Mm"`
+        echo "##[set-output name=repo_plus_tag;]$(echo ${REPO}:${TAG}-`date "+%m-%d-%y-%Hh-%Mm"`)"
```

Comment:
Can you change some of the dashes to help differentiate between all the numbers?  For example, instead of `gcc-8-10-13-20-16h-11m` maybe `gcc-8_10-13-20_16h-11m`?